src/UI/Component/Panel/Listing/Appointement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Interface Appointment
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Appointment extends \ILIAS\UI\Component\Component {
```

Comment:
No real preference here, just an observation from my side. Another docstring for that class would have been enough.